reporter/html.js

```diff
@@ -676,20 +676,20 @@ QUnit.log( function( details ) {
 	// Also, we need to check details.expected existence, as it can exist and be undefined
 	if ( !details.result && hasOwn.call( details, "expected" ) ) {
 		if ( details.negative ) {
-			expected = escapeText( "NOT " + QUnit.dump.parse( details.expected ) );
+			expected = "NOT " + QUnit.dump.parse( details.expected );
 		} else {
-			expected = escapeText( QUnit.dump.parse( details.expected ) );
+			expected = QUnit.dump.parse( details.expected );
 		}
 
-		actual = escapeText( QUnit.dump.parse( details.actual ) );
+		actual = QUnit.dump.parse( details.actual );
```

Comment:
> If it's not a fix, but a breaking change, we might consider deprecating it. (from the public api)

It's the correct fix for the reported issue, but it is also breaking because the current behavior is broken. We should be escaping the values we put into the returned markup rather than assuming the passed in values are correctly escaped.

I don't necessarily see a reason to remove the function from the public API, but we will probably want to deprecate the current behavior and move to the new behavior in 2.0.